### PR TITLE
New contribute page

### PR DESCRIPTION
**Pull request by mirshko**

### CODE REVIEW

**Comment by lucasvo** [comment]
Is this deployed anywhere so i can look at it?

File: src/components/Footer/index.js [hunk 1]
```diff
@@ -95,7 +95,7 @@ const Footer = () => (
               Documentation
             </FooterExternalLinkItem>
             <FooterExternalLinkItem href="/whitepaper">Whitepaper</FooterExternalLinkItem>
```
Comment (rdinicut) [inline src/components/Footer/index.js]: The name of this component seems verbose :).  Does it do more than set target="_blank"?

File: src/helpers/index.js [hunk 2]
```diff
@@ -7,4 +7,34 @@ const MEDIUM_CDN = "https://cdn-images-1.medium.com/max/720";
 
 const MEDIUM_URL = "https://medium.com/centrifuge/";
 
-export { lastInArray, MEDIUM_CDN, MEDIUM_URL, RichTextRenderer };
+const responsiveGrid = {
```
Comment (rdinicut) [inline src/helpers/index.js]: nice!

File: src/components/Footer/index.js (hunk 1 above)
Comment (rdinicut) [reply src/components/Footer/index.js]: I think you could have used the Anchor Comp here if you had the need for custom styling that is not generic the FooterComponent could style his children

Comment (mirshko) [reply src/components/Footer/index.js]: it also adds `rel="norefferrer nofollow"`

**Pull request merged**